packages/grafana-data/src/dataframe/DataFrameJSON.test.ts

```diff
@@ -0,0 +1,83 @@
+import { FieldType } from '../types/dataFrame';
+import { DataFrameJSON, dataFrameFromJSON } from './DataFrameJSON';
+
+describe('DataFrame JSON', () => {
+  describe('when called with a DataFrame', () => {
+    it('then it should reverse the order of values in all fields', () => {
```

Comment:
"should decode values not supported natively in JSON (e.g. NaN, Infinity)"

Reply:
```suggestion
    it('should decode values not supported natively in JSON (e.g. NaN, Infinity)', () => {
```